teams/infra/edit-and-sync-configurations-managed-by-argocd/README.md

```diff
@@ -0,0 +1,53 @@
+# edit-and-sync-configurations-managed-by-argocd
+
+This composite Github Action (GHA) is aimed to be used by the Camunda Infrastructure team to automate the update of Kubernetes resource configurations (source of truth) managed by ArgoCD.
```

Comment:
I'd like to think about it more, but my initial concerns are around breaking the UNIX (and GHA workflow steps) principle of "do one thing and do it well", while this actions is dealing with:
- check out a repo
- patch the files
- commit the changes
- trigger an argocd sync

not sure it creates more simplicity instead of going for plain github actions doing the same actions step by step.

for instance, `yq-yaml-processor` and `sync-argocd-applications` don't have this problem (they deal with one topic only: patching yaml or syncing ArgoCD apps)

Reply:
I agree, let's keep an eye on it to see how many use cases for such action would occur and whether it makes sense to generalize it in the future.